src/App.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Tabs from './components/Tabs';
+import './App.scss';
+
+/* eslint-disable */
+const tabs = [
+  { id: 'tab-1', title: 'Home', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Profile', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Contact', content: 'Some text 3' },
+];
+/* eslint-enable */
+
+class App extends React.Component {
+  state = {
+    selectedTabId: tabs[0].id,
+    selectedTitle: tabs[0].title,
```

Comment:
Store active tab in state and use any of its fields when necessary.
```suggestion
    selectedTab: tabs[0],
```